editor/keyboard.js

```diff
@@ -20,31 +20,3 @@ export function createKeyManager() {
     getKeyState,
   };
 }
-
-function createRotationManager() {
```

Comment:
@MiguelCastillo I removed this function because ESLint was warning about this fn not being used. Let me know if we need to revert this and I can turn off the rule here.

Reply:
nah its fine to remove. thanks for cleaning up.